cmd/modelcmd/controller.go

```diff
@@ -19,19 +19,18 @@ var (
 	// ErrNoControllersDefined is returned by commands that operate on
 	// a controller if there is no current controller, no controller has been
 	// explicitly specified, and there is no default controller.
-	ErrNoControllersDefined = errors.New(`no controller
+	ErrNoControllersDefined = errors.New(`No controllers registered.
 
-Please either create your own new controller using "juju bootstrap" or
-connect to another controller that you have been given access to using "juju register".
+Please either create a new controller using "juju bootstrap" or connect to
+another controller that you have been given access to using "juju register".
 `)
```

Comment:
I know .... I started to do that, but it was going to be too complicated of a change, when all that was really asked for was for the text to be tweaked.